.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
 - bundle exec rubocop
 - export RAILS_ENV=test
 - bundle exec rake db:create db:schema:load
```

Comment:
this `schema:load` change caused us fits. not committing schema.rb changes is now required. not sure why we weren't doing that before

I changed out the line below to keep testing more in-line with local dev process and the README